db/maintenance/info_test.go

```diff
@@ -0,0 +1,194 @@
+package maintenance
```

Comment:
suggestion: fix it

Reply:
I think the test names with the nested structure is a good representation enough but having a test table can make the tests a bit more clearer